src/header/common/pragma.rs

```diff
@@ -33,7 +33,6 @@ impl Header for Pragma {
         parsing::from_one_raw_str(raw).and_then(|s: String| {
             let slice = &s.to_ascii_lowercase()[..];
             match slice {
-                "" => None,
```

Comment:
Fixed in another commit. It should always be a `None` for single value headers, so I added it to `from_one_raw_str()`. List headers already check for empty values in a similiar way. This variant is now obsolete.